draft-ietf-rtcweb-jsep.xml

```diff
@@ -642,7 +642,9 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
           larger-than-expected number of ICE components is used, or
           because the pool has not had enough time to gather
           candidates, the remaining candidates are gathered as
-          usual.</t>
+          usual. This only occurs for the first offer/answer exchange,
+          after which the candidate pool is emptied and no longer used.
+          </t>
```

Comment:
> What's the reason for this?

To simplify the API, since we don't have a use case for using the pool mid-session (or at least, that's the conclusion I think we came to).

> it doesn't agree with the text below about setLocal() being the triggering element
The pool size is set in stone after setLocal(), but Cullen pointed out that if using a local pranswer, you don't want to discard pooled candidates quite yet, until a final answer is applied.